oonidata/db/create_tables.py

```diff
@@ -53,50 +55,69 @@ def typing_to_clickhouse(t: Any) -> str:
     if t == Outcome:
         return "String"
 
+    if t == dict[str, str]:
+        return "Map(String, String)"
+
     raise Exception(f"Unhandled type {t}")
 
 
-def create_query_for_observation(obs_class: Type[Observation]) -> str:
+def create_query_for_observation(obs_class: Type[Observation]) -> Tuple[str, str]:
     columns = []
     for f in fields(obs_class):
         type_str = typing_to_clickhouse(f.type)
-        columns.append(f"     `{f.name}` {type_str}")
+        columns.append(f"     {f.name} {type_str}")
 
     columns_str = ",\n".join(columns)
 
-    return f"""
+    return (
+        f"""
     CREATE TABLE {obs_class.__table_name__} (
 {columns_str}
     )
     ENGINE = ReplacingMergeTree
     ORDER BY (timestamp, observation_id, measurement_uid)
     SETTINGS index_granularity = 8192;
-    """
+    """,
+        obs_class.__table_name__,
+    )
 
-def create_query_for_verdict() -> str:
+
+def create_query_for_verdict() -> Tuple[str, str]:
     columns = []
     for f in fields(Verdict):
         type_str = typing_to_clickhouse(f.type)
-        columns.append(f"     `{f.name}` {type_str}")
+        columns.append(f"     {f.name} {type_str}")
```

Comment:
What is the impact of removing the backticks?

Reply:
It makes it possible to easily copy paste the create table commands into the CLI. If you have the backtick in them, the shell will interpret them as "execute the command called `f.name`".